src/index.js

```diff
@@ -31,26 +31,100 @@ function loadAppId(callback) {
     });
 }
 
+function getUrl() {
+    var server_url = localStorage.getItem('config.server_url') || 'frontend.binaryws.com';
+    return 'wss://' + server_url + '/websockets/v3';
+}
+
+function processRedirect(selected_language_name) {
+    loadAppId(function(err, app_id_json) {
+        var ip_country;
```

Comment:
```suggestion
        var client_country;
```
I think we can rename this as we are actually storing the client_country instead of the ip